querybook/webapp/const/datadoc.ts

```diff
@@ -93,3 +93,22 @@ export interface IDataDocDAGExport {
     dag: Record<string, Node[] | Edge[]>;
     meta: Record<string, any>;
 }
+
+export interface IDataDocDAGExporterData {
+    type: 'url' | 'text';
+    meta: IDataDocDAGExporterMetaOption;
+}
+
+export interface IDataDocDAGExporter extends IDataDocDAGExporterData {
+    name: string;
+}
+
+export type IDataDocDAGExporterMeta = Record<
+    string,
+    IDataDocDAGExporterMetaOption
+>;
+
+export interface IDataDocDAGExporterMetaOption {
```

Comment:
isn't meta a free form dictionary?

Reply:
oh this should be updated to match formfield